docs/source/examples/text.rst

```diff
@@ -9,107 +9,6 @@ Text
             text = Text('Hello world').scale(3)
             self.add(text)
 
```

Comment:
Here I would like to make a reference to https://manimce--525.org.readthedocs.build/en/525/reference/manim.mobject.svg.text_mobject.Text.html#manim.mobject.svg.text_mobject.Text , something like `:class:`~.Text`
@behackl , could you maybe once show me the exact syntax for that? :)

Reply:
I think adding references like that is a really good idea for now! You almost had it right, simply say

```
This is an example that illustrates how to use the :class:`~.Text` class.
```

(The surrounding text is just to illustrate how to use these references within a paragraph; you probably want to replace it by something that actually makes sense.)

Reply:
Cool.
Just added it :)